test/mocks/services/iLegacyCustomClusterClient.mock.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ILegacyCustomClusterClient } from 'opensearch-dashboards/server';
+const legacyClusterClientMock = jest.fn() as any;
```

Comment:
Can this be a regular object instead since we are adding properties inside it below?

Reply:
Works as an object as well, updated.